### ISSUE TITLE
Request: Remnant should pay for working Jump Drives

### ISSUE THREAD

**Issue by Bardagh** [endless-sky/endless-sky#5504]
**Related Issue Links**
Couldn't find any.

**Describe the solution you'd like**
They pay for broken jump drives; why not working ones?

**Describe alternatives you've considered**
This could be developed further into a story expansion where the Remnant develop their own jump drives. They of all humans have the most experience reverse-engineering alien tech, after all. If that were to happen, they could start raiding the Graveyard & maybe even colonize it via a series of missions wiping out or capturing & repurposing Ka'het factory stations.

Alternatively, enough jump drives sold might lead to them building a raiding fleet to capture more jump drives, snowballing into regular raids into Korath & Ka'het space & subsequent improvements to their ships using salvaged tech.

**Comment by Zitchas** [comment]
You can, actually, sell them working JD. They'll pay 250k for them in the outfitter. That being said, there's no specific "selling JD to the Remnant" mission for two reason:
1) From a lore/story point of view, broken JD that aren't catastrophically destroyed are much more rare than working JD. Usually when JD break, they do so in ways that utterly destroy the JD and often the ship they are on too. That being said, they are avid salvagers, and have been salvaging Korath ships since the very beginning, and thus they already have thousands of working JD in their possession. Along with that, they have had a comparatively very long time to research them, too. Broken JD are a relatively new phenomenon, and thus they are eager to study them.
2) From a meta perspective, once the player is routinely running around Remnant / EW space, there is an extremely high likelyhood that they have a JD. Unlike the Unfettered, which do not have a long storyline surrouding them, and thus JD equipped players aren't going to be overly bothered by frequent messages about "do you want to sell the unfettered your JD?", the player *is* expected to be spending a lot of time in Remnant territory, and frequent requests to sell their JD would get really annoying really fast.

So, given those two reasons, I decided to not have a routine message pestering the player to sell the Remnant their JD. The Remnant are, however, always happy to buy JD from the player via the outfitter. If we ever get the capability to set custom prices for outfits in different locations, the Remnant will definitely be paying more for working JD than the current 25%. (actually, Remnant will probably have higher purchase prices for most alien outfits)

That being said, given point 1, the Remnant *do* have scout ships out there with JD; and they have thoroughly explored much of the galaxy using them. As Dawn points out in the Astral missions that will hopefully be merged soon, the Remnant have conducted 43 scientific missions to Sagittarius A*, and have explored considerable amounts of the galaxy. Due to their rarity and how valuable they are, however, they only actually put JD on ships for specific missions, and do not have standing fleets that use them. (hence why you never see Remnant with JD).

(Just to prove a point about Remnant exploration capabilities, I once took off from Viminal in a starling with a JD, cloaked, then proceeded to explore the entire galaxy before returning to Viminal. For the entire duration of that exploration, I never uncloaked, never landed, and never received fuel from any other ship. It'd be hard to do it in a stock Starling, but in a properly outfitted starling it is actually quite easy - if time consuming.)

**Comment by Bardagh** [comment]
Anyone will pay 250k for a working jump drive. And they obviously have the ability to acquire broken jump drives, given how often they cripple Korath ships in their space for you to hunt down (bounty jobs). Korath ships with working jump drives are much rarer than those with broken ones in the Ember Waste, especially in Remnant space proper. The fact that they pay so well for broken ones to me just says they need a lot of them for research.

**Comment by Bardagh** [comment]
You could work around the engine constraints without pestering the player by offering jobs to deliver working gear (jump drives, syscores, maybe Ka'het weapons & cooling systems) to Remnant scientists.

I respect your reasons. I'm just trying to offer alternatives. Thank you for your work, and please don't take my arguments as an attack <3

**Comment by Hurleveur** [comment]
I guess this will resolve it then https://github.com/endless-sky/endless-sky/pull/6404
Since with that PR you can easily make all alien tech be worth twice their normal price.

**Comment by Zitchas** [comment]
Yes, yes it will. Another issue that will likely be able to be closed soon!

**Comment by ThrawnCA** [comment]
> From a lore/story point of view, broken JD that aren't catastrophically destroyed are much more rare than working JD. Usually when JD break, they do so in ways that utterly destroy the JD and often the ship they are on too.

That's a good point. If someone came across a piece of glass that had clearly been smashed, but the fragments were circular instead of sharp, that would be bizarre and warrant further investigation, in a way that an intact glass sculpture, however beautiful, would not.
